### PR TITLE
[RHPAM-740] fix for Kie server redeploy caused exception with PostgreSQL 9.6

### PR DESCRIPTION
**Pull request by calvinzhuca**
…jdbc RPM is installed

Signed-off-by: calvin zhu <calvinzhu@hotmail.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by calvinzhuca** [comment]
this is the jira link: 
https://issues.jboss.org/browse/RHPAM-740

These two PRs will provide temp fix by adding the latest PostgreSQL JDBC to replace the old 9.2 JDBC, before the latest RPM is installed. 
https://github.com/jboss-container-images/jboss-kie-modules/pull/21
https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/31

File: os.bpmsuite.executionserver/configure.sh [hunk 1]
```diff
@@ -34,6 +34,13 @@ mkdir -p ${KIE_DIR}/repository
 chown -R jboss:root ${KIE_DIR}
 chmod -R 777 ${KIE_DIR}
 
+#temp fix for https://issues.jboss.org/browse/RHPAM-740, before the latest PostgreSQL JDGC rpm is provided
+#only copy the JDBC if it exists
+FILE=${SOURCES_DIR}/postgresql-42.2.2.jar
```
Comment (errantepiphany) [inline os.bpmsuite.executionserver/configure.sh]: Sorry to be nit-picky, but can you please rename FILE to POSTGRESQL_JAR (or similar)? "FILE" just seems to generic.

**Comment by calvinzhuca** [comment]
no problem, will do the rename. Don't merge yet, I'm doing local testing. After it works, I will squash the commit.

File: os.bpmsuite.executionserver/configure.sh (hunk 1 above)
Comment (calvinzhuca) [reply os.bpmsuite.executionserver/configure.sh]: no problem, will do the rename. Don't merge yet, I'm doing local testing. After it works, I will squash the commit.

**Comment by errantepiphany** [comment]
@spolti , @calvinzhuca , @bmozaffa, @rcernich : Perhaps we create a follow-up jira that depends on a request for a redhat-signed jar? Because right now, there isn't one in brew: https://brewweb.engineering.redhat.com/brew/search?match=glob&type=maven&terms=postgresql-*.jar

**Comment by spolti** [comment]
+1

**Comment by calvinzhuca** [comment]
Updated with latest postgresql-42.2.2.redhat-2.jar to replace the downloaded community version.

**Pull request closed without merging**